util.js

```diff
@@ -105,11 +105,18 @@ function packageOrigin (env, pkg) {
   }
 }
 
+function localPrebuild (url, opts) {
+  var propName = 'npm_config_' + (opts.pkg.name || '').replace(/[^a-zA-Z0-9]/g, '_') + '_local_prebuilds'
```

Comment:
It might be worth extracting the common regex-based renaming logic between this and `getHostMirrorUrl` so they don't get out of sync with each other.